daemon/api_prompting.go

```diff
@@ -58,7 +58,7 @@ var (
 		POST:        postRules,
 		Actions:     []string{"add", "remove"},
 		ReadAccess:  interfaceOpenAccess{Interfaces: []string{"snap-interfaces-requests-control"}},
-		WriteAccess: interfaceAuthenticatedAccess{Interfaces: []string{"snap-interfaces-requests-control"}, Polkit: polkitActionManage},
+		WriteAccess: interfaceOpenAccess{Interfaces: []string{"snap-interfaces-requests-control"}},
```

Comment:
it's probably worth adding a comment here that postRules operates on per-user segregated rule sets?

Reply:
I really would change it to mention the the function:

// postRules